Userland/Services/WindowServer/KeymapSwitcher.h

```diff
@@ -27,15 +27,15 @@ class KeymapSwitcher final : public Core::Object {
 
     String get_current_keymap() const;
 
+    void setkeymap(AK::String const&);
```

Comment:
I know the API already existed, but since you're moving it, please also rename it to `set_keymap`.